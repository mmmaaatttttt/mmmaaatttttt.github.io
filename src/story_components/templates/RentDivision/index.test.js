import React from "react";
import { shallow } from "enzyme";
import toJson from "enzyme-to-json";
import { RentDivision } from ".";

describe("smoke and snapshot tests", () => {
  it("renders successfully", () => {
    shallow(<RentDivision />);
  });

  it("renders first demonstration successfully", () => {
    const wrapper = shallow(<RentDivision />);
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
