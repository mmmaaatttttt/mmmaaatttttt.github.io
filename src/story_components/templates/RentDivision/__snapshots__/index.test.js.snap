// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`smoke and snapshot tests renders first demonstration successfully 1`] = `
<div>
  <styled.div
    fullWidthAt="small"
    margin="0 auto"
    width="70%"
  >
    <LabeledSlider
      color="#FF8F34"
      fadeIcons={false}
      handleValueChange={[Function]}
      max={5}
      maxIcon="expand-arrows-alt"
      min={2}
      minIcon="compress-arrows-alt"
      sliderHeight={6}
      sliderPadding={10}
      step={1}
      tickCount={4}
      title="Mesh Size"
      value={4}
    />
    <styled.div
      column={false}
      cross="stretch"
      flex={1}
      main="center"
      margin="0"
      shouldWrap={false}
      textAlign="left"
      width="auto"
    >
      <Button
        color="white"
        disabled={false}
        large={false}
        onClick={[Function]}
      >
        Start Demonstration
      </Button>
    </styled.div>
  </styled.div>
  <styled.div
    fullWidthAt="small"
    margin="0 auto"
    width="55%"
  >
    <TooltipProvider
      render={[Function]}
    />
  </styled.div>
</div>
`;
